src/suggest-edit.js

```diff
@@ -145,11 +145,23 @@ function renderForm(schema, company_data = undefined) {
             }
         }
     });
+
+    // Create and render form
     bf = BrutusinForms.create(schema);
     bf.render(
         document.getElementById('suggest-form'),
         company_data || (PARAMETERS['name'] ? { name: PARAMETERS['name'] } : {})
     );
+
+    // Set attributes to avoid autocompletion on the form
+    // This is to avoid private data to be submitted (e.g. via a Password Manager)
+    document.querySelectorAll('.brutusin-form form').forEach((el) => {
```

Comment:
This selector returns nothing, if I am not mistaken. Did you mean `input`? As far as I understand the attributes should be set in the `input` field anyways.

Did you test this with a password manager? :) 

```suggestion
    document.querySelectorAll('.brutusin-form input').forEach((el) => {
```

Reply:
From the MDN documentation, they can also be set on the form and not repeated everywhere.
Maybe it's better to try this version, if it works for all as expected then it's better I think 🤔 

https://developer.mozilla.org/en-US/docs/Web/Security/Practical_implementation_guides/Turning_off_form_autocompletion
> You can do this either for an entire form, or for specific input elements in a form.

Reply:
Good find! I thought I saved and added that change to my commit, but I did not apparently 😄